book-src/ch02-00-let-the-code-begin.md

```diff
@@ -11,36 +22,48 @@ We'll start with the absolute basics: adding two numbers together.
 
 Already a lot going on, but don't worry.
 
+<!-- 
+   I removed this because the learner doesn't really know what interactive 
+   programming is or why that's exciting at this point. Comes a little out of nowhere?
+-->
 Firstly, we're already living up to the title of this book.
 In the block above, you'll see two panes:
-the top is editable code, and the bottom is the result.
-*Interactive programming, woo!*
+the top is editable code, and the bottom is the result. Here,
+we can interactively write some code, and the result will appear to us instantly!
 
-(That's nothing compared to what we'll be getting up to later, though.)
+(That's nothing compared to what we'll be getting up to later, though 😉)
 
 Second, that look awfully weird, doesn't it?
 Why the parentheses, *and why does the plus sign come before the numbers?*
 
-This is a particular feature of the family of languages that Clojure comes from.
-Clojure is composed of *expressions*.
-*Expressions* can either be an individual value like the number `42`
-or the word `"apples"`, or it could be a list of items, in which
-case those items are surrounded by parentheses.
-When an expression is a list of items
-surrounded by parentheses, the first item is treated as an *operator* to be
-applied to whatever follows.
-
-In this case, the plus sign is an addition operator.
-The numbers that follow are what get fed into that addition.
+<!-- 
+   Felt like this paragraph could use a rework. Hope you like the changes.
+ -->
+This is a peculiar feature of Clojure that might look a little odd at first glance!
+Here's what's going on - Clojure is composed of what we call *expressions*.
+*Expressions* are just anything that can be *evaluated* and in turn produce *values*. 
+An expression can be as simple as the number `42` or the word `"apples"`, 
+but they can also get more complicated, like a list of items. 
+When an expression is a list of items, we group those items together inside 
+parentheses - like in our list above `(+ 2 2)`. When our expression is a list of 
+items like this, we call the first element the *function* and everything after 
+it the *arguments* 
+(or think about math class, where we called these the *operator* and *operands*).
```

Comment:
I had trouble with this paragraph in particular. The wording here definitely flows much better than the original.

For this overall section, I'm debating whether to use the terms operator/operand at all, to avoid using potentially unnecessary jargon.

Reply:
I wanted to leave the terms in there to draw a parallel to concepts the reader is likely already familiar with, but I have no qualms about dropping the jargon either.